### PR TITLE
Parrot stress relief pull 2016

### PR DESCRIPTION
**Pull request by Incoming5643**
Buffs parrots health from 20 to 80.
Parrots have gone from opportune attackers (I attack weak babies) to aggressive idiots (attack so long as I have good health)
Fixes a obscure little bug where a parrot left alone without anyone to hear them couldn't speak through radios.
Ghost poly can now posses people who try to attack them and force them to spout parrot lines.
The ghost poly easter egg now has a very rare chance of happening every time poly dies.

All changes based on real world observations of parrot behaviors.

:cl:
add: Parrots have been made more robust.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -902,7 +906,7 @@
 /mob/living/simple_animal/parrot/Poly/death(gibbed)
 	if(!memory_saved)
 		var/go_ghost = 0
-		if(rounds_survived == longest_survival || rounds_survived == longest_deathstreak)
+		if(rounds_survived == longest_survival || rounds_survived == longest_deathstreak || prob(0.666))
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/parrot.dm]: :-1:

keep it as-is

Comment (optimumtact) [inline code/modules/mob/living/simple_animal/parrot.dm]: also make it a define so help me god save me from this madness

File: code/datums/diseases/parrotpossession.dm [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	spread_flags = SPECIAL
+	disease_flags = CURABLE
+	cure_text = "Holy Water."
+	cures = list("holywater")
+	cure_chance = 20
+	agent = "Avian Vengence"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	desc = "Subject is possesed by the vengeful spirit of a parrot. Call the priest."
+	severity = MEDIUM
+	var/mob/living/simple_animal/parrot/Poly/ghost/parrot
```
Comment (optimumtact) [inline code/datums/diseases/parrotpossession.dm]: what is this

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 3]
```diff
@@ -325,8 +329,8 @@
 
 			parrot_interest = user
 			parrot_state = PARROT_SWOOP
-			if (user.health < 50)
-				parrot_state |= PARROT_ATTACK //weakened mob? fight back!
+			if(health > 30) //Let's get in there and squawk it up!
```
Comment (optimumtact) [inline code/modules/mob/living/simple_animal/parrot.dm]: PARROT_SQUAWK_THRESHOLD

**Comment by Cheridan** [comment]
>Fixes a obscure little bug where a parrot left alone without anyone to hear them couldn't speak through radios.

if a parrot squawks in the forest, and nobody is around to hear...

File: code/modules/mob/living/simple_animal/parrot.dm (hunk 1 above)
Comment (Incoming5643) [inline code/modules/mob/living/simple_animal/parrot.dm]: As the parrots age and the records get higher the opportunities to see the ghost will only grow rarer and rarer.  prob(0.666) is pretty goddamn rare for a roll that usually only happens once every other round or so.

File: code/datums/diseases/parrotpossession.dm (hunk 2 above)
Comment (Incoming5643) [inline code/datums/diseases/parrotpossession.dm]: yes, it's an easter egg where a poly that is repeatedly slaughtered comes back as an unkillable ghost.

You know, for fun.

**Pull request merged**